### PR TITLE
Voting fix 2

### PR DESCRIPTION
**Pull request by korczis**
- voting.js was moved to own module - voting/
- references to this module were modified accordingly 
- any logic was not touched

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 685065288ca1fc1c32c84b9cceb44eea6bcae0e5

File: lib/routes/votes.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
     function RouteVotes(app) {
       app.get('/votes', function (req, res) {
         var data = {
-          votes: voting.currentVotes,
+          votes: voting.currentVotes
```
Comment (shalecraig) [inline lib/routes/votes.js]: :smile:

**Comment by shalecraig** [comment]
Despite the comment, I'm unsure about making such big changes, especially if there's no description other than "Voting fix 2"

**Comment by shalecraig** [comment]
:-1:

**Comment by avdg** [comment]
Better to keep moving and changing voting.js separate to prevent big diffs. :-1: for now

**Comment by korczis** [comment]
@avdg This is trying to move voting separate module ..

**Comment by avdg** [comment]
Meh... I've tried to diff both these files with an external tool: https://www.diffchecker.com/ebd78wty

**Comment by SpenserJ** [comment]
:+1:

**Comment by drsirmrpresidentfathercharles** [comment]
:+1:

**Comment by shalecraig** [comment]
Is the bot supposed to auto-close things like this?

**Comment by dbpokorny** [comment]
:-1: Travis CI just started complaining about merge conflicts, so please re-roll

**Comment by dos1** [comment]
>This is trying to move voting.js into separate module ..

In node, files *already are* separate modules.

**Comment by pandzieciol** [comment]
:+1:

**Comment by plor** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
#### :-1: The vote failed. This PR will now be closed.

----
**Tallies:**
:+1:: 4 (50%) 
:-1:: 4 (50%)

These users aren't stargazers, so their votes were not counted: 
 * @botwillacceptanything

**Pull request closed without merging**